app/client/src/components/shared/DateSlider.js

```diff
@@ -66,11 +66,20 @@ const trackStyles = {
   margin: '0.5em 0',
 };
 
-function DateSlider({ bounds, disabled, onChange }) {
-  const [range, setRange] = useState(bounds);
+function DateSlider({ disabled, min, max, onChange }) {
+  const [minYear, setMinYear] = useState(0);
+  const [maxYear, setMaxYear] = useState(100);
```

Comment:
```suggestion
  const [minYear, setMinYear] = useState(min);
  const [maxYear, setMaxYear] = useState(max);
```

You can just initialize the minYear/maxYear values with the min/max.